sssd_test_framework/roles/ipa.py

```diff
@@ -154,13 +164,37 @@ def test_example(client: Client, ipa: IPA):
         """
         return self._password_policy
 
+    @property
+    def naming_context(self) -> str:
+        """
+        LDAP  suffix.
+        """
+        if len(self.host.domain.split(".")) < 1:
+            raise IndexError("Second level domain missing!")
+
+        tld = self.host.domain.split(".")[-1]
+        sld = self.host.domain.split(".")[-2]
+        return f"dc={sld},dc={tld}"
```

Comment:
This works now, but it may not always be the case. It would be better to find basedn in /etc/ipa/default.conf

Reply:
Ack, done.